docs/src/administrator/manual/django-admin.md

```diff
@@ -57,20 +57,193 @@ After successfully changing the password, the administrator will receive a confi
+
+    - **Draft species uploads**: The `Draft species uploads` table is used for storing the draft species uploads records. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing draft species uploads.
+
+    - **Layers**: The `Layers` table is used for storing the layers records. The administrators can modify individual layers within the application, which includes defining their display properties, filter settings, and their relationships with context layers. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing layers.
+
+    - **Map sessions**: The `Map sessions` table is used for storing the map session records. The administrators can monitor and configure these sessions. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing map sessions.
+
+    - **Spatial data**: The `Spatial data` table is used for storing the spatial data records. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing spatial data.
+
+    - **Statistical models**: The `Statistical models` table is used for storing the statistical model records. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing statistical models.
+
+![Django Admin Tables](./img/admin-13.png)
+
+8. **NOTIFICATION**: The administrators can use this section to perform operations related to reminders.
+
+    - **Reminders**: The `Reminders` table is used for storing the reminder records. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing reminders.
+
+9. **OCCURRENCE**: The administrators can use this section to perform operations related to occurrences.
+
+    - **Survey methods**: The `Survey methods` table is used for storing the survey methods records. In the field of wildlife and environmental studies, survey methods are essential for systematically collecting data about the occurrence and distribution of various species and ecosystems. These methods are carefully designed to provide accurate information for research, conservation, and management purposes. This table provides administrators with the capability to perform various operations, including adding, deleting, updating, and viewing survey methods.
+
+10. **OTP STATIC**: The administrators can use this section to perform operations related to static OTP devices.
```

Comment:
Not every one knows what the acronym stands for so explaining it once is required.
```suggestion
10. **OTP STATIC**: The administrators can use this section to perform operations related to static OTP (one-time pin) devices.
```

Reply:
okay